go.mod

```diff
@@ -108,12 +108,13 @@ require (
 	github.com/natefinch/atomic v1.0.1
 	github.com/oklog/run v1.1.0
 	github.com/okta/okta-sdk-golang/v2 v2.20.0
-	github.com/openbao/go-kms-wrapping/v2 v2.3.0
+	github.com/openbao/go-kms-wrapping/v2 v2.3.1-0.20250321181437-1a1c5b0c79b1
```

Comment:
@cipherboy is that OK, or should I wait for a new release of `go-kms-wrapping` ?

Reply:
@phsym This is fine, it is part of [our release process](https://openbao.org/docs/policies/release/) to cut a new tag for g-k-w.